components/email/src/Email.svelte

```diff
@@ -577,7 +580,6 @@
         message: message,
         thread: activeThread,
         value,
```

Comment:
`focus_body_onload` is prop of composer, its value shouldn't be passed from Mailbox or Email component.